plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/polygon/PolygonColor/index.tsx

```diff
@@ -21,42 +15,27 @@ const PolygonColor: React.FC<BaseFieldProps<"polygonColor">> = ({
   dataID,
   value,
   editMode,
-  isActive,
   onUpdate,
 }) => {
   const [items, updateItems] = useState(value.items);
 
-  const handleMoveUp = useCallback(
-    (idx: number) => {
-      updateItems(c => {
-        const newItems = moveItemUp(idx, c) ?? c;
-        onUpdate({
-          ...value,
-          items: newItems,
-        });
-        return newItems;
-      });
-    },
-    [value, onUpdate],
-  );
+  const handleMoveUp = useCallback((idx: number) => {
+    updateItems(c => {
+      const newitems = moveItemUp(idx, c) ?? c;
+      return newitems;
+    });
+  }, []);
 
-  const handleMoveDown = useCallback(
-    (idx: number) => {
-      updateItems(c => {
-        const newItems = moveItemDown(idx, c) ?? c;
-        onUpdate({
-          ...value,
-          items: newItems,
-        });
-        return newItems;
-      });
-    },
-    [onUpdate, value],
-  );
+  const handleMoveDown = useCallback((idx: number) => {
+    updateItems(c => {
+      const newitems = moveItemDown(idx, c) ?? c;
+      return newitems;
+    });
+  }, []);
 
   const handleAdd = useCallback(() => {
     updateItems(c => {
-      const newItem: { condition: Cond<any>; color: string } = {
+      const newPointColor: { condition: Cond<any>; color: string } = {
```

Comment:
Since you updated the name here to newPointColor, please do for the remaining callbacks to stay consistent

Reply:
Done.